Libplanet/Store/IStore.cs

```diff
@@ -244,5 +244,16 @@ void PutBlock<T>(Block<T> block)
         long CountTransactions();
 
         long CountBlocks();
+
+        /// <summary>
+        /// Forks transaction nonces from
+        /// <paramref name="sourceChainId"/> to
+        /// <paramref name="destinationChainId"/>.
+        /// </summary>
+        /// <param name="sourceChainId">The chain ID of transaction nonces to
+        /// fork.</param>
+        /// <param name="destinationChainId">The chain ID of destination
+        /// transaction nonces.</param>
+        void ForkTxNonces(Guid sourceChainId, Guid destinationChainId);
```

Comment:
Of course, I know there is a test to check it works in `BlockChainTest` but could you add unittest for this new method in `StoreTest`?

Reply:
I'll add.

Reply:
[Added!](https://github.com/planetarium/libplanet/pull/1198/files#diff-4682d77dc711a5c0e7ec81aaeeb20519c75c475bd24ccfbfa3d6caec762caff1)